lib/isogram_cases.rb

```diff
@@ -0,0 +1,21 @@
+class IsogramCase < OpenStruct
+  def name
+    str_to_ascii = input.downcase.unpack('U*').select { |c| c < 128 }.map(&:chr)
+    joined_string = str_to_ascii.join.gsub(/[ -]/, '_')
+    'test_%s' % joined_string
+  end
+
+  def do
```

Comment:
Can you think of a better name for this? 
Having methods with the same name as keywords should be avoided if possible.